DevAcademy/Services/MapService.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  MapService.swift
+//  DevAcademy
+//
+//  Created by Tomáš Melichar on 27.08.2023.
+//
+
+import Foundation
+import MapKit
+
+final class MapService {
```

Comment:
Rekl bych, ze tahle service uplne neni potreba nebo ma mozna trosku zavadejici pojmenovani. Protoze ty funkce az tak nesouvisi s mapou, nebo spis muzou byt v budoucnu volany klidne i z jinych obrazovek, protoze jde spise o otevirani ruznych URL.

Bud bych tu Service prejmenoval na napr. URLService, nebo bych ty funkce treba presunul do extension nad URL a pouze je volal z ViewState. 🤔 

Ale ber to jenom jako subjektivni pripominku, klidne to muzes nechat i takhle. 😉

Reply:
Skvělý tip. Souhlasím, dal jsem to do URL jako `static func`, zatím jsem neměnil názvy funkcí :) 

A v poho, myslel jsem si, že máte hromadu práce. :)